app/services/geek_joke.service.js

```diff
@@ -0,0 +1,15 @@
+const axios = require('axios');
```

Comment:
remember that the file name should be plural `weets.js`

Reply:
file renamed